### ISSUE TITLE
Accessibility issues via voice control

### ISSUE THREAD

**Issue by shanaqui** [HabitRPG/habitrpg-ios#957]
"Recently, I’ve had to use accessibility features on my iPad to click things due to an arm injury. However, right now when I enable voice control, all of the numbers that appear in overlay are set to open the task into edit mode, rather than check it off as a default. This means that to check off a task, I have to use the mouse grid mode, which usually requires saying a series of five different numbers to check off a single task. It would be great if there could be a number overlays that would automatically check off the task, and then a different set of numbers to edit the task. That would save a lot of time!"

> iOS Version: 14.3
> Device: iPad Pro (10.5-inch)
> App Version: 3.1.1 (526)
> User ID: 7bde7864-ebc5-4ee2-a4b7-1070d464cdb0
> Level: 68
> Class: warrior
> Is in Inn: true
> Uses Costume: true
> Custom Day Start: 6

**Comment by saraolson** [comment]
fix for now would be to make checking off a task the default action rather than opening the task edit screen

**Comment by rilygreat** [comment]
> The following lines help us find and squash the Bug you encountered. Please do not delete/change them.
> iOS Version: 15.6
> Device: iPhone14,3
> App Version: 3.5.4 (694)
> User ID: 3eaa4d67-f70c-438c-bdf8-2a2ac926c00c
> Level: 312
> Class: warrior
> Is in Inn: false
> Uses Costume: true
> Custom Day Start: 4
> 
> Please describe the bug you encountered:
> Unable to check off second checklist item with Voiceover enabled. Not sure if this applies to all checklists but this list only activates the daily, bringing up the menu to change the whole daily.

**Comment by EricMiddelhove** [comment]
@saraolson Is there already an idea on how to open the edit screen then? 

Kind regards
Eric

**Comment by saraolson** [comment]
@EricMiddelhove im not too familiar with the number overlays referenced in the original report here:

> It would be great if there could be a number overlays that would automatically check off the task, and then a different set of numbers to edit the task. That would save a lot of time!"


but i think their solution would be the intended idea? have the first number be check off the task, and the 2nd edit, as editing a task is the secondary action. expanding a checklist would be a tertiary action, etc. once again, im not super familiar with this way of interacting, but their suggestion sounds valid.

**Comment by EricMiddelhove** [comment]
Ok I'll have a look at it :)

**Comment by saraolson** [comment]
after lots of app updates and changes to this screen, we need to do another pass and see what's still needed for this issue. once we do that we can update with more specifics of what we'd like to change.

if anyone uses voice control and would like to add their feedback on that, please do!

**Comment by saraolson** [comment]
We're going to close this in favor of creating new issues for any specific bugs or improvements with Voice Over in the future 😄
